charts/modernization-api/values-dts1.yaml

```diff
@@ -33,6 +33,8 @@ reportingService:
   enabled: "true"
   personReportingServicePort: 8091
   organizationReportingServicePort: 8092
+  investigationReportingServicePort: 8093
+  postProcessingReportingServicePort: 8095
```

Comment:
In order to prevent other environment from breaking you will also need to add lines 36-37 to all other values-*.yaml (to be safe).

Reply:
Updated in all other places. Thanks for pointing out.